#message file for manual control
int16 x
int16 y
int16 z
int16 r
